Matchismo/Model/CardMatchingGame.m

```diff
@@ -46,36 +44,56 @@ - (instancetype)initWithCardCounts:(NSUInteger)count usingDeck:(Deck *)deck
 static const int MATCH_BONUS = 4;
 static const int COST_TO_CHOOSE = 1;
 
-- (void)chooseCardAtIndex:(NSUInteger)index {
+- (NSString *)chooseCardAtIndex:(NSUInteger)index
```

Comment:
The signature of this method do not suppose to return any values. Let's leave it as `void` for now

Reply:
I have a guess but that didn't work 😭

Reply:
Needs some senior magic obviously

Reply:
![image](https://github.com/khaletska/Matchismo/assets/13601748/8dc6a0fd-681d-4594-a461-d5b08f976895)
😉
Ping if you have any help with that